src/taipy/rest/api/resources/datanode.py

```diff
@@ -71,17 +74,20 @@ class DataNodeResource(Resource):
                 properties:
                   datanode: DataNodeSchema
         404:
-          description: datanode does not exist
+          description: Data node does not exist
```

Comment:
No data node has the *datanode_id* identifier.
or
No such data node.

(as well as similar phrasing in all the files)